contracts/stellar-axelar-operators/src/migrate.rs

```diff
@@ -0,0 +1,25 @@
+use soroban_sdk::{Address, Env, Vec};
+
+mod storage {
+    use soroban_sdk::{contracttype, Address};
+    use stellar_axelar_std::contractstorage;
+
+    #[contractstorage]
+    #[derive(Clone, Debug)]
+    pub enum MigrationDataKey {
+        #[instance]
+        #[status]
+        Operators { account: Address },
+    }
+}
+
+pub type MigrationData = Vec<Address>;
+
+pub fn migrate(env: &Env, migration_data: MigrationData) {
+    for account in migration_data {
+        if storage::is_operators(env, account.clone()) {
+            super::storage::set_operator_status(env, account.clone());
+            storage::remove_operators_status(env, account);
+        }
+    }
```

Comment:
are we planning to change the operator set at the same time as migrating keys?

Reply:
No, it should just migrate existing ones. I'm fine with converting this to use `ensure!(storage::is_operators(..))` and failing instead